mtags/src/main/scala/scala/meta/internal/mtags/ScalaToplevelMtags.scala

```diff
@@ -159,7 +159,15 @@ class ScalaToplevelMtags(
           acceptTrivia()
           val name = newIdentifier
           withOwner(currRegion.owner) {
-            term(name.name, name.pos, Kind.OBJECT, 0)
+            term(
+              name.name,
+              name.pos,
+              // hack: (exclusively) making symbol kind of extension methods to Kind.Method
+              // so that we can tell it's an extension method in Indexer.scala
+              // TODO: add properties that represents "extension" method to semanticdb schema
+              Kind.METHOD,
+              0
```

Comment:
set extension methods symbol kind to `METHOD` to distinguish between extension methods and others.

It would be better if we can attach some properties that represents this is "extension" method, but there's no properties for it in SemanticDB https://scalameta.org/docs/semanticdb/specification.html#symbolinformation
Maybe we should add `extension` property to SemanticDB ?🤔  

what do you think about it @tgodzik ?